manifests/config.pp

```diff
@@ -7,27 +7,27 @@
 class auditd::config {
   assert_private()
 
-  if $::auditd::default_audit_profile != undef {
+  if $auditd::default_audit_profile != undef {
```

Comment:
Since this is a major version bump, should we get rid of this deprecation logic?